packages/table-core/src/utils/getSortedRowModel.ts

```diff
@@ -18,8 +18,9 @@ export function getSortedRowModel<TData extends RowData>(): (
         const sortedFlatRows: Row<TData>[] = []
 
         // Filter out sortings that correspond to non existing columns
+        const columns = table._getAllFlatColumnsById();
```

Comment:
I think we want to avoid the getAll call here, and simply just add the optional chaining down below maybe?

Reply:
We're using `table.getColumn` which internally uses `table._getAllFlatColumnsById()` anyway. It's a cache value and accessing its value is O(1) operation so it may be just fine?